### PR TITLE
Fix datapoints for the dahsboard

### PR DESCRIPTION
**Pull request by avinal**
- remove active datapoints

### CODE REVIEW

File: cmd/web/handlers.go [hunk 1]
```diff
@@ -25,11 +25,10 @@ import (
...
 )
 
-
 // Declare wrap as a wrapper type to the db.Application type to be used in the HomePage.
 type wrap db.Application
 
-// HomePage function parses the templates and then feeds it with the data we 
+// HomePage function parses the templates and then feeds it with the data we
 // need to display.
 func (app *wrap) HomePage(w http.ResponseWriter, r *http.Request) {
```
Comment (ramessesii2) [inline cmd/web/handlers.go]: HomePage can be only a func now, (app *wrap) cab be totally removed.

File: cmd/web/main.go [hunk 2]
```diff
@@ -42,12 +46,12 @@ func main() {
...
-	app := (*wrap)(dapp)
+	rClient = db.GetRedisClient(&cred)
+	// dapp, err := db.GetValues(rClient)
+	// if err != nil {
+	// 	log.Fatalf("Couldn't get the values from the source")
+	// }
+	// app := (*wrap)(dapp)
 
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", app.HomePage)
```
Comment (ramessesii2) [inline cmd/web/main.go]: HomePage can be a func now, rather than a method

**Review by ramessesii2 — CHANGES_REQUESTED** [review]

**Review by ramessesii2 — APPROVED** [review]
lgtm

**Review by UtkarshChaurasia — APPROVED** [review]
lgtm

**Pull request merged**